spec/integration/node/integration.spec.js

```diff
@@ -73,6 +78,24 @@ describe('integration', () => {
           done()
         })
       })
+
+      it('no keep alive', (done) => {
```

Comment:
To me this sounds like a "describe" block since it is describing more of the _context_ when it should state what it is _testing_.

Suggestion (pseudo code):
```ts
describe("without keep alive", () => {
  beforeEach(() => {
        const httpAgent = new HttpAgent({ keepAlive: false })
        configs.gatewayConfigs.HTTP.configure = () => ({ agent: httpAgent })
  })
  it("does not reuse the socket but only attaches listeners once to http agent `sockets`", () => ...
```

(please reword appropriately since I am not 100% sure what is tested)